spec/mailers/previews/supervisor_mailer_preview_spec.rb

```diff
@@ -3,6 +3,7 @@
 
 RSpec.describe SupervisorMailerPreview do
   let!(:supervisor) { create(:supervisor) }
+  let!(:volunteer) { create(:volunteer) }
```

Comment:
Can you make it explicit that the volunteer's supervisor is the supervisor created above? Might get flaky in the future if we don't.

Reply:
Thank you @schoork for the feedback. I have addressed the reported point and pushed the fix. Docker check is failing due to other issue, can you trigger it again and review the changes again to merge it.